tests/testthat/test-pkg-arrow.R

```diff
@@ -480,3 +480,63 @@ test_that("arrow translation for sign() works", {
     tibble::tibble(dbl = c(-1, -1, -1, 0, 1))
   )
 })
+
+test_that("arrow translation for %in% works", {
+  skip_if_not(has_arrow_with_substrait())
+
+  # TODO: fix "Can't create substrait.Expression from object of type 'NULL'"
+  # case of zero items
```

Comment:
Internally, in the Arrow `%in%` function, despite being identical to the DuckDB `%in% function at present, `lhs` is a Substrait Expression, whereas `rhs` is coming through as `NULL`.  In DuckDB, they are both Substrait Expressions. Perhaps something to do with how empty inputs are handled by each?

Reply:
Thanks!